doc/classes/Viewport.xml

```diff
@@ -188,6 +188,18 @@
 		<member name="disable_3d" type="bool" setter="set_disable_3d" getter="is_3d_disabled" default="false">
 			Disable 3D rendering (but keep 2D rendering).
 		</member>
+		<member name="fsr_upscale_custom_quality" type="float" setter="set_fsr_upscale_custom_quality" getter="get_fsr_upscale_custom_quality" default="1.0">
+			Manually sets the internal scale factor for FidelityFX Super Resolution 1.0.
+		</member>
+		<member name="fsr_upscale_mipmap_bias" type="float" setter="set_fsr_upscale_mipmap_bias" getter="get_fsr_upscale_mipmap_bias" default="0.0">
+			Affects the final texture sharpness. Negitive values make textures sharper, while positive values make textures blurrier.
+		</member>
+		<member name="fsr_upscale_quality" type="int" setter="set_fsr_upscale_quality" getter="get_fsr_upscale_quality" enum="Viewport.FSRUpscaleQuality" default="0">
```

Comment:
Docs need to be updated now that we aren't using ``fsr_upscale_quality``

Reply:
Removed